README.md

```diff
@@ -44,12 +44,11 @@ HTTP Status code of 200 is returned with the file when `:fileNameAndPath` exists
 
 ### Notes for Developer updating the files to the latest version
 
-- Current Version of EU requirements `V2.21`
+- Previous Version of EU requirements `V2.21` (Feb 2022), Current Version of EU requirements (March 2022)
```

Comment:
remove Feb2022 we received it way before that

Reply:
This would be in reference to the version 2 EU requirements. -> `V2.21`
The version 1 requirements would have their own versioning (I think the KEL mentioned V1.24) and I'm not sure what the V1 minor version is